File: code/intermediate/polymorphism/methodOverriding/MethodOverriding.java
package code.intermediate.polymorphism.methodOverriding;

class Parent {
    void printInfo() {
        System.out.println("Parent Info");
    }
}

class MethodOverloading extends Parent {
    void printInfo() {
        System.out.println("Child Info");
    }
    public static void main(String args[]){
        MethodOverloading mo = new MethodOverloading();
        mo.printInfo();
    }
}

File: code/intermediate/polymorphism/methodOverriding/MethodOverriding.js
function Parent() {}

Parent.prototype.printName = function () {
    console.log('Parent print info');
}
function Child() {}
Child.prototype = Object.create(Parent.prototype);
Child.prototype.constructor = Child;
Child.prototype.printName = function () {
    console.log('Child print info');
}

const child = new Child();
child.printName();